src/main/java/com/example/tournamentMaker/team/Team.java

```diff
@@ -16,6 +17,7 @@
 @NoArgsConstructor
 @Getter
 @Setter
+@EqualsAndHashCode(of = "id")
```

Comment:
https://youtu.be/UPWkpl5PL_w 
Obejrzyj zrozum i mi odpisz dlaczego robienie equals i hashcode w oparciu o id jest gownianym pomyslem

Reply:
Bo może to prowadzić do problemów, ponieważ podczas tworzenia encji "id" jeszcze nie ma